searx/engines/google.py

```diff
@@ -76,9 +76,9 @@
     'SE': 'www.google.se',  # Sweden
     'TH': 'www.google.co.th',  # Thailand
     'TR': 'www.google.com.tr',  # Turkey
-    'UA': 'www.google.com.ua',  # Ikraine
-    # 'CN': 'www.google.cn',  # China, only from china ?
-    'HK': 'www.google.com.hk',  # Hong kong
+    'UA': 'www.google.com.ua',  # Ukraine
+    # 'CN': 'www.google.cn',  # China, only from China ?
```

Comment:
More for information than anything else : actually google.cn contains only a link to google.com.hk which is not accessible from main land china. So comment could be something like : "empty web site".